kola/tests/misc/state.go

```diff
@@ -22,6 +46,16 @@ func init() {
 		// This test is normally not related to the cloud environment
 		Platforms: []string{"qemu", "qemu-unpriv"},
 	})
+	register.Register(&register.Test{
+		Run:         OsReset,
+		ClusterSize: 1,
+		Name:        "cl.osreset.ignition-rerun",
+		UserData:    ignitionRerun,
+		Distros:     []string{"cl"},
+		MinVersion:  semver.Version{Major: 3511},
```

Comment:
Should the minversion be higher?

```suggestion
		MinVersion:  semver.Version{Major: 3530},
```